### PR TITLE
feat(va-tabs): support hidden pagination

### PR DESCRIPTION
**Pull request by Valar103769**
closes https://github.com/epicmaxco/vuestic-ui/issues/2469

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/ui/src/components/va-tabs/VaTabs.vue [hunk 1]
```diff
@@ -106,6 +107,7 @@ export default defineComponent({
     right: { type: Boolean, default: false },
     center: { type: Boolean, default: false },
     grow: { type: Boolean, default: false },
+    withPagination: { type: Boolean, default: true },
```
Comment (m0ksem) [inline packages/ui/src/components/va-tabs/VaTabs.vue:110]: Better rename it to `hide-pagination` prop, so it is `false` by default.

```vue
<va-tabs hide-pagination />
```
vs

```vue
<va-tabs :with-pagination="false" />
```

File: packages/ui/src/components/va-tabs/VaTabs.vue [hunk 2]
```diff
@@ -472,6 +481,10 @@ export default defineComponent({
     .va-tabs__tabs {
       position: absolute;
       height: 100%;
+
```
Comment (m0ksem) [inline packages/ui/src/components/va-tabs/VaTabs.vue:484]: Might be easier to use `v-if="showPagination && !$props.hidePagination"` on buttons in component template.

**Review by m0ksem — CHANGES_REQUESTED** [review]
Thanks. Looks good, but it can be a bit simpler.

File: packages/ui/src/components/va-tabs/VaTabs.vue (hunk 2 above)
Comment (Valar103769) [reply packages/ui/src/components/va-tabs/VaTabs.vue:484]: new pr: https://github.com/epicmaxco/vuestic-ui/pull/2605
 ```css
 .va-tabs__tabs {
      position: absolute;
}
```
`position:absolute` need to be remove, let width grow by content

**Pull request closed without merging**